aurorastation.dme

```diff
@@ -10,6 +10,7 @@
 #define DEBUG
 // END_PREFERENCES
 // BEGIN_INCLUDE
+#include "_maps\sccv_horizon.dm"
```

Comment:
Why? I don't think this is intentional?

Reply:
had that activated for testing, yes, was accidental